usecase/interactor/poke_interactor.go

```diff
@@ -0,0 +1,37 @@
+package interactor
+
+import (
+	"github.com/ToteEmmanuel/academy-go-q12021/domain/model"
+	"github.com/ToteEmmanuel/academy-go-q12021/usecase/presenter"
+	"github.com/ToteEmmanuel/academy-go-q12021/usecase/repository"
+)
+
+type pokeInteractor struct {
+	PokeRepo      repository.PokeRepository
```

Comment:
It's ok to define these interfaces here on the same package you're planning to use them

Reply:
Did some refactoring on the interfaces.